### PR TITLE
Design polish

### PR DESCRIPTION
**Pull request by jrubenoff**
Lots of little fixes all around. Not done yet, but pretty proud of this for about six hours' work.

Unfortunately, didn't do too much with questions in this PR. The interactions there need to be a bit more complex, I think.

### CODE REVIEW

**Comment by ajb** [comment]
Cool, this is definitely going in the right direction.

I'm not clear on whether you wanted this PR to be merged before you're done or not, but I'm going to list the stuff in here that would prevent it from being merged:

img

img

img

img

img (Not a regression, but really needs fixing.)

img

img

img

img
- The conditionals for the "Post an opportunity" button in the nav bar, as well as the "sign in" link disappearing when the user is on the "sign in" page, are too jarring. It's much better to just leave the links in so that the nav bar does not change abruptly.

File: config/locales/devise.en.yml [hunk 1]
```diff
@@ -35,17 +35,17 @@ en:
       updated: "Your password has been changed successfully. You are now signed in."
       updated_not_active: "Your password has been changed successfully."
     registrations:
-      destroyed: "Bye! Your account has been successfully cancelled. We hope to see you again soon."
-      signed_up: "Welcome! You have signed up successfully."
+      destroyed: "Your account has been cancelled. We hope to see you again soon."
+      signed_up: "Thanks for signing up! Let's start "
```
Comment (ajb) [inline config/locales/devise.en.yml]: Huh?

File: config/locales/devise.en.yml [hunk 2]
```diff
@@ -3,9 +3,9 @@
 en:
   devise:
     confirmations:
-      confirmed: "Your email address has been successfully confirmed."
-      send_instructions: "You will receive an email with instructions for how to confirm your email address in a few minutes."
-      send_paranoid_instructions: "If your email address exists in our database, you will receive an email with instructions for how to confirm your email address in a few minutes."
+      confirmed: "Your account is now confirmed! Let's start looking at opportunities."
```
Comment (ajb) [inline config/locales/devise.en.yml]: This translation is used for vendors _and_ staff. This is not correct.

File: config/locales/devise.en.yml [hunk 3]
```diff
@@ -3,9 +3,9 @@
...
+      send_instructions: "We sent you an email with a link to confirm your account. It might take you a few minutes to receive it."
```
Comment (ajb) [inline config/locales/devise.en.yml:7]: If we're going to modify this translation, let's go ahead and remove the "few minutes".

File: config/locales/en.yml [hunk 4]
```diff
@@ -2,51 +2,51 @@
 en:
   access_denied: Sorry, you are not allowed to access that page.
   admin: Admin
-  allow_vendors_to_ask_questions: Allow vendors to ask questions publicly about this opportunity
+  allow_vendors_to_ask_questions: Let vendors ask public questions about this opportunity
+  answered_by_user_at_time: Answered by %{user} at %{timestamp}
```
Comment (ajb) [inline config/locales/en.yml]: Timestamps look like this: "4/12/16 at 3:15pm"

So that's why we use "on".

File: config/locales/en.yml [hunk 5]
```diff
@@ -2,51 +2,51 @@
...
   attachments: Attachments
+  browse: Browse
   browse_opportunities: Browse opportunities
-  clear_filters: Clear filters
+  clear: Clear
   confirm_alert: Are you sure?
-  contact_info: Contact info
-  create_account: Create account
+  contact: Contact
+  create_account: Sign up
```
Comment (ajb) [inline config/locales/en.yml]: This should be a new i18n key.

File: config/locales/en.yml [hunk 6]
```diff
@@ -2,51 +2,51 @@
...
-      cta: Subscribe for business opportunities
-      hint: Sign up to be notified when new projects are posted
+      cta: Sign up today
+      hint: Browse new business opportunities, and get notified when new ones are posted.
       hint_signed_in: New opportunities are being added every day!
       title: Win contracts with the City of Philadelphia
     recent_opportunities:
-      browse_all: Browse all
-      title: Recent opportunities
+      see_more: See more...
```
Comment (ajb) [inline config/locales/en.yml]: I've been using a unicode ellipsis here: `…`

File: config/locales/en.yml [hunk 7]
```diff
@@ -89,56 +87,63 @@ en:
...
   sign_in: Sign in
   sign_out: Sign out
   simple_form:
     hints:
       opportunity:
         category_ids: |-
-          Vendors who subscribe to these categories will be notified of your opportunity.
-          Choose these carefully!
-        contact_info: This information will be publicly displayed on your opportunity page
+          Your choice of categories will affect the vendors that see your opportunity.
```
Comment (ajb) [inline config/locales/en.yml]: This seems to infer that vendors can only see opportunities that are in their categories.

How about s/see/are notified of/?

File: config/locales/en.yml [hunk 8]
```diff
@@ -89,56 +87,63 @@ en:
...
-        name: Your full name, please!
-        password: Make it at least 8 characters
+        password: Make it at least 8 characters.
     labels:
       opportunity:
-        category_ids: Categorize as…
+        category_ids: Categories
         publish_at: Publish on…
-        question_dates: Question period from…
+        question_dates: Accept questions between...
```
Comment (ajb) [inline config/locales/en.yml]: Same goes for this ellipsis, and any following ones.

Comment (ajb) [inline config/locales/en.yml]: (Control + Command + Space is handy for inserting these.)

File: config/locales/simple_form.en.yml [hunk 9]
```diff
@@ -9,7 +9,7 @@ en:
       # When using html, text and mark won't be used.
       # html: '<abbr title="required">*</abbr>'
     error_notification:
-      default_message: "Please review the problems below:"
+      default_message: "Please fix the errors below."
```
Comment (ajb) [inline config/locales/simple_form.en.yml:12]: This should never actually be displayed. Did you see it somewhere?

**Comment by ajb** [comment]
Also, I must add: this really needs to be merged ASAP because of how massive it is -- it basically prevents me from doing work elsewhere in the repo.

**Comment by ajb** [comment]
Homepage centering is not mobile-friendly:

img

**Comment by ajb** [comment]
"Recent opportunities" blank slate:

img

File: config/locales/devise.en.yml (hunk 3 above)
Comment (jrubenoff) [inline config/locales/devise.en.yml:7]: The goal is to prevent them from pressing the "Resend email" link 10 seconds later.

**Comment by jrubenoff** [comment]
Everything you mentioned has been fixed, except the conditionals in the navbar. To me, keeping the links around would be jarring. Both pages are modal, in the sense that the user must do something on that page before leaving. The link disappears once you've clicked it to indicate you are now in that mode.

Also, having a big orange button for "Post this opportunity" when you're already on that page is super distracting.

If it really bothers you, though, you're welcome to undo it.

**Comment by ajb** [comment]
You got it all! Thanks again for jumping on this.
- The nav conditions: I don't mind 'em for now. However, it really bugs me that the nav "jumps up" when going from the home page to any other page. I understand that the home page looks better with some padding on the top of the nav bar, but this _really_ irks me. I went ahead and removed the `0.5rem` padding on the top of the navbar.
- Please note that we do not use `%{thing}` interpolation in i18n any longer. (It's not safe for translation.)
- Page header font family changes on mobile, and I'm not sure if it's a bug or not: http://take.ms/JBLBS. I'll open an issue.

**Pull request merged**